drivers/regulator/regulator_npm1300.c

```diff
@@ -73,6 +73,11 @@ enum npm1300_gpio_type {
 #define LDSW2_SOFTSTART_MASK  0x30U
 #define LDSW2_SOFTSTART_SHIFT 4U
 
+struct npm1300_gpio_info {
+	uint8_t pin;
+	uint8_t flags;
```

Comment:
should this use the gpio typedef? `gpio_pin_t` and `gpio_dt_flags_t`

Reply:
I don't believe so, the property has no relation to the devicetree GPIO API beyond re-using the `GPIO_ACTIVE_LOW` define.